pkgs/by-name/ol/ollama/package.nix

```
@@ -180,10 +181,15 @@ goBuild {
   );
 
   preBuild = ''
-    # disable uses of `git`, since nix removes the git directory
-    export OLLAMA_SKIP_PATCHING=true
     # build llama.cpp libraries for ollama
-    go generate ./...
+    make -j 5
+  '';
+
+  postInstall = lib.optionalString stdenv.hostPlatform.isLinux ''
+    # copy libggml_*.so and runners into lib
+    # https://github.com/ollama/ollama/blob/v0.4.1/llama/make/gpu.make#L90
```

Maybe the link should be updated?

```suggestion
    # copy libggml_*.so and runners into lib
    # https://github.com/ollama/ollama/blob/v0.4.3/llama/make/gpu.make#L90
```